### DISCUSSION TITLE
docker hub

### DISCUSSION THREAD

**Discussion by chenhaoenen** [AbdBarho/stable-diffusion-webui-docker#228] [Q&A]
Thanks a lot for a great article, I cant build images from Dockerfile, because of the network, Is there a corresponding docker hub link?

**Comment by AbdBarho** [comment]
@chenhaoenen Unfortunately, there is no prebuilt docker image.

In fact, even a pre built image would still require internet for additional models download.

**Reply by ndm13** [reply]
Deploying a single stack (or image) and not needing to clone a repo in order to do setup, when Dockerfiles are built for doing that one time setup, feels ideal here. I use GitHub Actions to publish Dockerfiles as images in the GitHub Container Repository and it has worked very well for me. Maybe consider doing that if you don't want to use Docker Hub?

**Comment by ovizii** [comment]
I'm also looking for the same! Not really looking to clone a repo, keep it updated and rebuild the image whenever there's an update. Would love to find a prebuilt image I can just pull from a registry.

**Reply by AbdBarho** [reply]
I truly agree, and I want this, it makes it easier for everyone, however, due to license constrains I am not allowed to publish container, since I am effectively publishing someone else's code.

**Reply by ovizii** [reply]
Oh, that's not good then. I won't ask you about the details as I assume you have already checked the situation and come to this conclusion.

**Comment by DevilaN** [comment]
Can it be solved by preparing base image with preinstalled packages from `requirements.txt` and other system packages that are needed to run SD and are not restricted by distribution license? Then local build process would result only in downloading other parts, but build process would be far more easier having ready to start with image.
